src/subscript/interp_relperm/interp_relperm.py

```diff
@@ -156,23 +152,9 @@ def _is_valid_interpolator(interp: dict):
     return valid
 
 
-@configsuite.validator_msg("Valid table entries")
+@configsuite.validator_msg("Low-base-high provided")
 def _is_valid_table_entries(schema: dict):
-
-    valid = False
-    try:
-        if schema["low"]:
-            valid = True
-    except (KeyError, ValueError, TypeError):
-        pass
-
-    try:
-        if schema["high"]:
-            valid = True
-    except (KeyError, ValueError, TypeError):
-        pass
-
-    return valid
+    return "base" in schema and "low" in schema and "high" in schema
```

Comment:
I guess the need for the entire function disappeared, since base, low and high now can be set to required in the schema.

Reply:
The same goes for the function below ..

Reply:
The validator for low-base-high was not even active! I will keep it in, since I need it in the next PR. 

family validator was not active either, now fixed. Did not find an enumeration class in configsuite, so I guess it should not be dropped?

Reply:
Ups, it should have been active. Well spotted!

Reply:
Yes, but https://github.com/equinor/subscript/pull/385/commits/c48cbfd3f46c4ddff71c4049dadf0b4da7d4ba02#diff-1cb24c63cb076e87801721182f5481b715e6763706cc6e1dd266b9f0e1ad6ce4R171 does the trick. 

Will add tests for this in next PR.

Reply:
(you might be right, before this PR it might have error gracefully after schema validation if low/high/base was missing).

Reply:
Fixes and tests are now applied to this PR.